packages/server/src/validators/BaseValidator.js

```diff
@@ -1,3 +1,5 @@
+import { isAddress } from 'web3-utils'
```

Comment:
This comment is maybe coming too late, but I question the design of these `Validator`s and their abstraction mechanisms:

1. The state inside (`this.errors`) seems to be reset on every validation, leading me to believe we must be reusing the same validator instances?
1. `return resetErrors()`'s name and behaviour are opaque to readers who do not understand the base class's implementation

Perhaps it'd be simpler to just export one or more validation functions for each concern, or compose a `Validator` with a stateful `ValidationError` class?

Reply:
created issue #197